src/pages/atomicdex/api/v20-dev/non_fungible_tokens/get_nft_transfers/index.mdx

```diff
@@ -0,0 +1,125 @@
+| token\_address     | string  | Address of token transferred.                                                                                                                                         |
+| token\_id          | string  | Token ID.                                                                                                                                                             |
+| possible\_spam     | boolean | If `true`, the contract address has [been identified](https://docs.moralis.io/web3-data-api/evm/nft-spam-detection) as associated with spam or suspicious activities. |
+| possible\_phishing | boolean | If `true`, the token has been identified as associated with phishing, as at least one of domain fields is found in database with phishing domains.                    |
+| transaction\_hash  | string  | Transaction ID of the transfer.                                                                                                                                       |
+| transaction\_index | string  | May be `null`. Transaction index of the transfer.                                                                                                                     |
+| log\_index         | string  | Log index of the transfer.                                                                                                                                            |
+| value              | string  | May be `null`. Tranfer value.                                                                                                                                         |
+| transaction\_type  | string  | May be `null`. Transaction type. Possible values are `Single`.                                                                                                        |
+| from\_address      | string  | Address of previous owner which sent the token(s).                                                                                                                    |
+| to\_address        | string  | Address of new owner which received the token(s).                                                                                                                     |
+| status             | string  | Transfer status. Will be either `Send` or `Receive`. When the `from_address` and `to_address` are the same (i.e. sending to yourself), this value will be `Receive`.  |
+| verified           | integer | May be `null`. A deprecated field which will be removed in future.                                                                                                    |
+| operator           | string  | May be `null`.                                                                                                                                                        |
```

Comment:
need to add two more fields

```rust
pub(crate) fee_details: Option<EthTxFeeDetails>,
pub(crate) confirmations: u64,
 ```
 
 ```rust
 pub struct EthTxFeeDetails {
    pub coin: String,
    pub gas: u64,
    /// WEI units per 1 gas
    pub gas_price: BigDecimal,
    pub total_fee: BigDecimal,
}
```
gas - the total amount of gas used for a transaction
gas_price - gas price
total_fee - the total cost of the transaction

UPD: `fee_details` in get transfers is actually `WithdrawFee` object.
But it is optional, as we can get None in eth api response for old transfers ([see PR description](https://github.com/KomodoPlatform/komodo-defi-framework/pull/1989))

Reply:
Added

Reply:
Oops, sorry, missed the part that in `get_nft_transfers` there is no `type` field, as we have in `WithdrawFee`, so fee info for `get_nft_transfers` will be just that

```
| Parameter       | Type           | Description                                                                       |
| --------------- | -------------- | --------------------------------------------------------------------------------- |
| coin            | string         | The coin which will be used to pay the transaction fee.                           |
| gas             | integer        | The amount of gas to be used for the transaction.                                 |
| gas\_price      | numeric string | Price per unit of gas to be used for the transaction.                             |
| total\_fee      | numeric string | Gas price multiplied by gas amount.                                               |
```
This object can be located below `NftTransfer` object

Reply:
Oops, sorry, missed the part that in `get_nft_transfers` there is no `type` field, as we have in `WithdrawFee`, so fee info for `get_nft_transfers` will be just that

```
| Parameter       | Type           | Description                                                                       |
| --------------- | -------------- | --------------------------------------------------------------------------------- |
| coin            | string         | The coin which will be used to pay the transaction fee.                           |
| gas             | integer        | The amount of gas to be used for the transaction.                                 |
| gas\_price      | numeric string | Price per unit of gas to be used for the transaction.                             |
| total\_fee      | numeric string | Gas price multiplied by gas amount.                                               |
```
This object can be located in `dex/api/v20/index.mdx`with other nft objects like `TokenMetadata`, `TokenTransferFilter` etc